.travis.yml

```diff
@@ -13,4 +13,4 @@ install:
   - travis_retry composer install
 
 script:
-  - phpunit
\ No newline at end of file
+  - phpunit -c phpunit.xml --tap --coverage-text
```

Comment:
I will make this a separate PR.

Why should there not be a phpunit config for our preferred testing settings, especially for CI? Also, why would we not want to run coverage on all commits? I feel that constantly monitoring test coverage is important, especially while the project is under active restructuring like this.